storybook/src/foundation/design-tokens/colour.docs.mdx

```diff
@@ -3,78 +3,129 @@
 import tokens from "@amsterdam/design-system-tokens/dist/index.json";
 import { Meta } from "@storybook/blocks";
 import { ColorPalette } from "../../docs/components/ColorPalette";
-import { StatusBadge } from "../../docs/components/StatusBadge";
 
 <Meta title="Brand/Design Tokens/Colour" />
 
-<StatusBadge description="The set of available colours and their names will change." />
-
 # Colour
 
 Our colour palette is vibrant and crisp.
 The main colours are red, blue, black, and white.
-They are complemented by a range of accent colors and shades of gray.
+They are complemented by a range of accent colors.
+
+## Palette
+
+The primary appearance of each color has been called level 60.
+Some colors offer a darker shade at level 80 for hover effects.
+A light neutral color is at level 20, white is at level 0, and black is at level 100.
 
 <ColorPalette>
-  <ColorPalette.Section>
-    <ColorPalette.Tile color={tokens.ams.color["primary-white"]} name="white" />
-    <ColorPalette.Tile color={tokens.ams.color["primary-red"]} name="red" />
-    <ColorPalette.Tile color={tokens.ams.color["primary-black"]} name="black" />
-  </ColorPalette.Section>
-  <ColorPalette.Section>
-    <ColorPalette.Tile color={tokens.ams.color["green"]} name="green" />
-    <ColorPalette.Tile color={tokens.ams.color["dark-green"]} name="dark green" />
-    <ColorPalette.Tile color={tokens.ams.color["blue"]} name="blue" />
-    <ColorPalette.Tile color={tokens.ams.color["primary-blue"]} name="dark blue" />
-    <ColorPalette.Tile color={tokens.ams.color["dark-blue"]} name="darker blue" />
-  </ColorPalette.Section>
-  <ColorPalette.Section>
-    <ColorPalette.Tile color={tokens.ams.color["yellow"]} name="yellow" />
-    <ColorPalette.Tile color={tokens.ams.color["orange"]} name="orange" />
-    <ColorPalette.Tile color={tokens.ams.color["magenta"]} name="magenta" />
-    <ColorPalette.Tile color={tokens.ams.color["purple"]} name="purple" />
-  </ColorPalette.Section>
-  <ColorPalette.Section>
-    <ColorPalette.Tile color={tokens.ams.color["neutral-grey1"]} name="light grey" />
-    <ColorPalette.Tile color={tokens.ams.color["neutral-grey2"]} name="grey" />
-    <ColorPalette.Tile color={tokens.ams.color["neutral-grey3"]} name="dark grey" />
-  </ColorPalette.Section>
+  <ColorPalette.Swatch name="red">
```

Comment:
If you remove this solution from brand color and jump straight to semantic color, I would think the docs describe the colors the same way.

Reply:
Oh wait, I assume you suggest not documenting the colour palette at all because we don’t use that structure in our brand tokens. See my other comment – I did consider it, but presenting a general colour palette is what people expect, and it creates a context for grasping our semantic tokens.